packages/fxa-settings/src/pages/Signin/SigninRecoveryCode/index.tsx

```diff
@@ -104,49 +91,19 @@ const SigninRecoveryCode = ({
           </p>
         </FtlMsg>
 
-        <form
-          noValidate
-          className="flex flex-col gap-4"
-          onSubmit={handleSubmit(onSubmit)}
-        >
-          {/* Using `type="text" inputmode="numeric"` shows the numeric pad on mobile and strips out whitespace on desktop. */}
-          <FtlMsg id="signin-recovery-code-input">
-            <InputText
-              type="text"
-              inputMode="numeric"
-              label="Enter 10-digit backup authentication code"
-              onChange={(e) => {
-                setRecoveryCode(e.target.value);
-                // clear error tooltip if user types in the field
-                if (recoveryCodeErrorMessage) {
-                  setRecoveryCodeErrorMessage('');
-                }
-              }}
-              onFocusCb={onFocusMetricsEvent ? onFocus : undefined}
-              errorText={recoveryCodeErrorMessage}
-              autoFocus
-              pattern="\d{10}"
-              className="text-start"
-              anchorStart
-              autoComplete="off"
-              spellCheck={false}
-              prefixDataTestId="signin-recovery-code"
-              required
-              tooltipPosition="bottom"
-              maxLength={10}
-            />
-          </FtlMsg>
+        <FormVerifyCode
+          {...{
+            formAttributes,
+            viewName,
+            email,
+            onSubmit,
+            code,
+            setCode,
+            codeErrorMessage,
+            setCodeErrorMessage,
+          }}
+        />
 
-          <FtlMsg id="signin-recovery-code-confirm-button">
-            <button
-              type="submit"
-              id="use-logged-in"
-              className="cta-primary cta-xl"
-            >
-              Confirm
-            </button>
-          </FtlMsg>
-        </form>
         <div className="mt-5 link-blue text-sm flex justify-between">
           <FtlMsg id="signin-recovery-code-back-link">
             <a href="/signin_totp_code">Back</a>
```

Comment:
Oh shoot I missed this before! This should be a `<Link>` element!

Reply:
Nice catch! Thank you